### PR TITLE
fix(DatePicker): add return object for onBlur

### PR DESCRIPTION
**Pull request by joakbjerk**
Adds a return object for the onBlur event, to make it possible to extract the date for the event object.

### CODE REVIEW

File: packages/dnb-eufemia/src/components/date-picker/DatePicker.d.ts [hunk 1]
```diff
@@ -247,8 +255,11 @@ export interface DatePickerProps
   /**
    * Will be called once the input loses focus.
    */
-  onBlur?: (event: React.FocusEventHandler<HTMLInputElement>) => void;
+  onBlur?: (
+    event: DatePickerEvent<React.FocusEvent<HTMLInputElement>>
+  ) => void;
 }
+
 export default class DatePicker extends React.Component<
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/date-picker/DatePicker.d.ts:263]: Should probably not be there 😉

**Review by tujoworker — APPROVED** [review]
Nice catch! Thanks for the tests too 🙏

File: packages/dnb-eufemia/src/components/date-picker/DatePicker.d.ts (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/components/date-picker/DatePicker.d.ts:263]: Ahh. You saw it already 👍

Comment (joakbjerk) [reply packages/dnb-eufemia/src/components/date-picker/DatePicker.d.ts:262]: Ahh, thought you meant the space, silly Github 😄

**Pull request merged**